start_production.sh

```diff
@@ -18,6 +18,19 @@ usage() {
   exit 1
 }
 
+# check all AWS_BUCKET env variables - if not present we should bail
+check_var() {
+  if [ -z $1 ]; then
+    echo "Did not find a required environment variable"
```

Comment:
I'd prefer a period at the end of the sentence. This is extremely minor in some ways, but it leads one to worry less that a copy/paste error has gotten only part of the error message.

Reply:
Oh, and while you're there, I'd very much prefer to see the `$1` as `"$1"`. This is a bug that is probably gone in modern bash, but it used to be that if you had a null string there, it got parsed as if you wrote `if [ -z ]; then`. Only by adding doublequotes as in `if [ -z "$1" ]; then` did you get `if [ -z "" ]; then` after interpolation. This also caused weird problems when the first character of the expansion was a "-" and bash thought it was another boolean/algebraic operator. I think these problems are gone away by now, and yet it triggers a fear in my brain and aesthetically still doesn't look as clean without the quotes.

Reply:
Probably a good idea to fix either way, will do.